modules/helm-creating-credentials-and-certificates-to-add-helm-repositories.adoc

```diff
@@ -0,0 +1,87 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/helm/configuring-custom-helm-chart-repositories.adoc
+
+[id="creating-credentials-and-certificates-to-add-helm-repositories_{context}"]
+= Creating credentials and CA certificates to add Helm chart repositories
+
+Some Helm chart repositories need credentials and custom Certificate Authority (CA) certificates to connect to it. You can use the web console as well as the CLI to add credentials and certificates.
+
+.Procedure
+To configure the credentials and certificates, and then add a Helm chart repository using the CLI:
+
+. In the `openshift-config` namespace, create a `configmap` with a custom CA certificate in PEM encoded format, and store it under the `ca-bundle.crt` key within the configmap.
++
+[source,terminal]
+----
+$ oc create configmap helm-ca-cert \
+--from-file=ca-bundle.crt=/path/to/certs/ca.crt \
+-n openshift-config
+----
++
+. In the `openshift-config` namespace, create a `secret`  to add the Client TLS Configurations. The client certificate and key must be in PEM encoded format and stored under the keys `tls.crt` and `tls.key`, respectively.
```

Comment:
Take this one or leave it :)

I see why you structured this one with the detail sentence following the action step (hence the take it or leave it) but for consistency, consider putting the command immediately following the action. E.g., 

. In the `openshift-config` namespace, create a `secret`  to add the Client TLS Configurations:
<command>
The client certificate and key must be in PEM encoded format and stored under the keys `tls.crt` and `tls.key`, respectively.